### ISSUE TITLE
Bungee issues

### ISSUE THREAD

**Issue by x1p** [RockinChaos/ItemJoin#330]
## Thanks for taking the time to open an assistance request!

I updated the hub with itemjoin 5.0.7
I'm facing 2 weird things.
- the bungee placeholders `%bungee_total%` and `%bungee_servername%` return 0 as a static number instead of the actual number of the amount of players on that server/total.
- chestcommands refuses to execute the bungee command to go to a certain server.
Both don't seem to be related to itemjoin. But when I switch back to 5.0.6, it's works flawless as ever.

All that ItemJoin have to do is clear inventory and give a compass. 
I'm in the woods here. I just need a clue in what direction to look for. What does itemjoin do to interfere with bungee things?

**Pastebin**
[_hub_5.0.7.zip](https://github.com/RockinChaos/ItemJoin/files/5181876/_hub_5.0.7.zip)

**Please complete the following information:**
 - 5.0.7 (v5.0.7-RELEASE-b537 according to log)
 - 1.16.1

**Additional context**

**Comment by RockinChaos** [comment]
So I was unable to replicate this as it is working perfectly fine for me in both v5.0.6 and v5.0.7 releases.
I also wasn't having any issues with the place holders--I did notice something weird with PlaceholderAPI though but after updating everything works as intended.

In-case this is an ItemJoin related bug I did some tweaks to the Bungee Class though I am not exactly sure what could be causing issues. All ItemJoin does that is Bungee related is sending players to servers, which is exactly what ChestCommands does.

Here is the latest snapshot that "potentially" will resolve your issues; http://ci.craftationgaming.com/job/ItemJoin/542/

**Comment by x1p** [comment]
I'm testing continuously to find out what could be the cause. I suspect a conflict somewhere.

**Comment by x1p** [comment]
I think the datatag setting is the cause
It seems to work when I change the default
```yaml
Settings:
  HeldItem-Slot: 4
  HeldItem-Triggers: JOIN, WORLD-SWITCH
  DataTags: true
```
Into:
```yaml
Settings:
  HeldItem-Slot: 4
  HeldItem-Triggers: DISABLED
  DataTags: false
```

**Comment by RockinChaos** [comment]
Is this with data tags enabled--and does this occur if just one item has inventory-modify or a specific one?

Thanks for being patient with me, this is quite difficult to figure out when I can't replicate it.

**Comment by RockinChaos** [comment]
Resolved; http://ci.craftationgaming.com/job/ItemJoin/543/
Bug was related to a custom event I added called PlayerPickItemEvent--was added to stop item duplication using the pick block "middle mouse" button.

**Comment by x1p** [comment]
> Is this with data tags enabled--and does this occur if just one item has inventory-modify or a specific one?
Yes there is only 1 item

Build 343 works indeed 👍

**Comment by RockinChaos** [comment]
This bug has been officially fixed in the release of ItemJoin v5.1.0.
You can grab it here; Download: https://www.spigotmc.org/resources/itemjoin.12661/download?version=356084

Please read the changelog as the documentation has not been updated yet;
https://github.com/RockinChaos/ItemJoin/wiki/Recent-Changes
